deployment/ansible/otp.yml

```diff
@@ -7,5 +7,4 @@
       apt: update_cache=yes
 
   roles:
-    - { role: "cac-tripplanner.otp"}
```

Comment:
I try not to create duplicates if a role is already listed as a dependency because it can cause the steps to get run multiple times (it'll be an OK instead of a CHANGED, but that still takes up some amount of time).